wolfcrypt/src/integer.c

```diff
@@ -716,11 +724,13 @@ int mp_read_unsigned_bin (mp_int * a, const unsigned char *b, int c)
 
 #ifndef MP_8BIT
       a->dp[0] |= *b++;
-      a->used += 1;
+      if (a->used == 0)
+          a->used = 1;
```

Comment:
Is gets 1 correct?  What if c was 128 and now we're touching up the final element.  Wouldn't a->used be much bigger than 1?

Reply:
never mind, looks good.